sound/soc/sof/intel/hda-dai-ops.c

```diff
@@ -237,6 +237,36 @@ static unsigned int generic_calc_stream_format(struct snd_sof_dev *sdev,
 	return format_val;
 }
 
+static unsigned int dmic_calc_stream_format(struct snd_sof_dev *sdev,
```

Comment:
@plbossart @juimonen the DMIC ops patch is not upstream yet. So this needs to be a fixup and squashed with it right?